### PR TITLE
Update readme to show compilation

### PR DESCRIPTION
**Pull request by mattdangerw**

### CODE REVIEW

**Review by chenmoneygithub — APPROVED** [review]

**Comment by mattdangerw** [comment]
Thanks! Just FYI, we should not merge this until after the public release.

**Review by jbischof — APPROVED** [review]
Thanks @mattdangerw!

**Pull request closed without merging**
